pcm_calculateG.m

```diff
@@ -1,10 +1,11 @@
-function [G,dGdtheta] = pcm_calculateG(M,theta)
-% function [G,dGdtheta] = pcm_calculateG(M,theta)
+function [G,dGdtheta] = pcm_calculateG(M,theta,G_hat)
```

Comment:
Added G_hat varargin for 'noiseceiling' case. I added this here instead of including it in the "if" loop in pcm_fitModelCrossval to make it easier for users to follow along.